src/InterNations/Component/HttpMock/Expectation.php

```diff
@@ -55,6 +56,28 @@ public function methodIs($matcher)
         return $this;
     }
 
+    public function headerExists($name)
+    {
+        $this->appendMatcher(
+            $this->matcherFactory->closure(function (Request $request) use ($name) {
+                return $request->headers->has($name);
+            })
+        );
+
+        return $this;
+    }
+
+    public function headerIs($name, $value)
+    {
+        $this->appendMatcher(
+            $this->matcherFactory->closure(function (Request $request) use ($name, $value) {
+                return $request->headers->contains($name, $value);
```

Comment:
Internally this does an in_array() check without the third parameter begin set to true, so it’s `==` not `===`. Can you rather do `in_array($value, $request->headers->get($name))`?